tests/antithesis/xsvm/main.go

```diff
@@ -157,12 +163,19 @@ func (w *workload) run(ctx context.Context) {
 }
 
 func (w *workload) confirmTransferTx(ctx context.Context, tx *status.TxIssuance) {
+	confirmed := true
 	for _, uri := range w.uris {
 		client := api.NewClient(uri, w.chainID.String())
 		if err := api.WaitForAcceptance(ctx, client, w.key.Address(), tx.Nonce); err != nil {
 			log.Printf("worker %d failed to confirm transaction %s on %s: %s", w.id, tx.TxID, uri, err)
-			return
+			confirmed = false
+			break
```

Comment:
Should we just `assert.Unreachable` here and `assert.Reachable` at the bottom and then remove the control flow change + `confirmed` bool?